tests/unit/helpers/format-html-message-test.js

```diff
@@ -32,8 +32,26 @@ test('exists', function(assert) {
 test('invoke the formatHTMLMessage directly', function(assert) {
   assert.expect(1);
   assert.equal(
-    service.formatHtmlMessage("<strong>Hello {name}</strong>", { name: "<em>Jason</em>" }),
-    "<strong>Hello &lt;em&gt;Jason&lt;/em&gt;</strong>"
+    service.formatHtmlMessage("<strong>Hello {name} {count, number}</strong>", {
+      name: "<em>Jason</em>",
+      count: 42000
+    }).string,
+    "<strong>Hello &lt;em&gt;Jason&lt;/em&gt; 42,000</strong>"
+  );
+});
+
+test('invoke the formatHTMLMessage directly with inlined locale', function(assert) {
```

Comment:
Fail test. `locale` would have never have been threaded through to Intl MessageFormat